.github/workflows/build.yaml

```diff
@@ -14,15 +14,13 @@ concurrency:
   group: ${{ github.head_ref }}-build
   cancel-in-progress: true
 jobs:
-  build:
-    name: docker
```

Comment:
I'd stick with the existing naming.

Reply:
Then there could be duplicate names, thus I switched.